plugins/inputs/sysstat/sysstat.go

```diff
@@ -0,0 +1,319 @@
+// +build linux
+
+package sysstat
+
+import (
+	"bufio"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+var (
+	firstTimestamp time.Time
+	execCommand    = exec.Command // execCommand is used to mock commands in tests.
+	dfltActivities = []string{"DISK"}
+)
+
+const parseInterval = 1 // parseInterval is the interval (in seconds) where the parsing of the binary file takes place.
+
+type Sysstat struct {
+	// Sadc represents the path to the sadc collector utility.
+	Sadc string `toml:"sadc_path"`
+
+	// Sadf represents the path to the sadf cmd.
+	Sadf string `toml:"sadf_path"`
+
+	// Activities is a list of activities that are passed as argument to the
+	// collector utility (e.g: DISK, SNMP etc...)
+	// The more activities that are added, the more data is collected.
+	Activities []string
+
+	// Options is a map of options.
+	//
+	// The key represents the actual option that the Sadf command is called with and
+	// the value represents the description for that option.
+	//
+	// For example, if you have the following options map:
+	//    map[string]string{"-C": "cpu", "-d": "disk"}
+	// The Sadf command is run with the options -C and -d to extract cpu and
+	// disk metrics from the collected binary file.
+	//
+	// If Group is false (see below), each metric will be prefixed with the corresponding description
+	// and represents itself a measurement.
+	//
+	// If Group is true, metrics are grouped to a single measurement with the corresponding description as name.
+	Options map[string]string
+
+	// Group determines if metrics are grouped or not.
+	Group bool
+
+	// DeviceTags adds the possibility to add additional tags for devices.
+	DeviceTags map[string][]map[string]string `toml:"device_tags"`
+	tmpFile    string
+	interval   int
+}
+
+func (*Sysstat) Description() string {
+	return "Sysstat metrics collector"
+}
+
+var sampleConfig = `
+  ## Path to the sadc command.
+  sadc_path = "/usr/lib/sa/sadc" # required
```

Comment:
can you put in here a comment noting that the debian default path is `/usr/lib/sysstat/sadc`?